docs/source/quick-start.rst

```diff
@@ -14,6 +14,42 @@ osm files to extract useful network information. As a result, the first step is
 Thanks to the open-source nature of OpenStreetMap, there are lots of APIs and mirror sites that we can use to
 download osm map data. We list several popular sites here for users to choose.
 
+0) osm2gmns built-in class: OSM_RelationID_Finder
+Get the relation id of a place of interest, eg. "Arizona State University", "Arizona, United States", "Tempe, AZ"...
+
+-- code-block:: python
+
+    >>> import osm2gmns as og
+    >>> rel = og.OSM_RelationID_Finder("Arizona State University")
```

Comment:
It's better to provide an API function to users rather than expose class constructors. please add one API function with the naming style of 'doSomeThing', like getNetFromFile, outputNetToCSV... also add function descriptions for that function. function descriptions will automatically appear in the user's guide function section. https://osm2gmns.readthedocs.io/en/latest/functions.html

Reply:
Thank you jiawei, I replaced original class-based: OSM_Relation_Finder to function-based: getOSMRelationID.
I updated readme and quick start as well.